src/tooltips.tsx

```diff
@@ -242,6 +243,19 @@ export class Tooltips {
 
 		document.body.appendChild(tooltipRoot);
 
+		// This is a special case for handling disabled elements. When a tooltip is
+		// shown on a tooltip element, the flag is activated and the next time this
+		// event listener is called the tooltip is hidden. This happens because
+		// a mouseover event will be triggered when the cursor leaves a disabled
+		// element.
+		// See: https://github.com/facebook/react/issues/4251#issuecomment-334266778
+		document.addEventListener('mouseover', () => {
```

Comment:
We should also remove the listener at some point.
Do we need to also check the origin of the event?
We could probably add a single event listener for all tooltip components, on the first `componentDidMount()` call or something, and then pub-sub to the rest of them.

Reply:
The `Tooltips` class is a singleton and only sets up a single event listener, so AFAIK this is not a concern